sda-commons-server-testing/README.md

```diff
@@ -70,6 +70,22 @@ public class CustomIT {
 }
 ```
 
+### SystemPropertyRule
+
+The [`SystemPropertyRule`](src/main/java/org/sdase/commons/server/testing/SystemPropertyRule.java)
+allows for overriding or unsetting system properties for (integration) tests and resets them to their original value when the tests have finished.
+
+To use the rule register it to your test class via the JUnit4 `@Rule`:
+
+```java
+@Rule
+public SystemPropertyRule PROP =
+  new SystemPropertyRule()
+    .setProperty(ENV_TO_SET, VALUE)
+    .setProperty(ENV_TO_SET_SUPPLIER, () -> VALUE)
+    .unsetProperty(ENV_TO_UNSET);
```

Comment:
"ENV_" also seems to be a relict from the EnvironmentRule. Can you change the code example? Either use matching constant names or choose any examples Strings that make sense here.